css/base-sass.css

```diff
@@ -5080,7 +5080,7 @@ blockquote:before {
       display: block; } }
 
 .minus {
-  background: url(../img/minus.png) 5px 12px no-repeat !important;
```

Comment:
@punalsc I think we should address this with specificity rather than an `!important` rule. I also wonder if `expanded` might be a more descriptive name than `minus`.